### DISCUSSION TITLE
Source Code

### DISCUSSION THREAD

**Discussion by davidlas** [nasko222/the-minecraft-recreation-project#157] [General]
I was wondering, since I barely know basic some coding (on paper) and even less of GitHub, if you have the source files of the remakes/original files that you used to base the remakes on? I figured you would based on what GH is. I believe I've said before I want to use the code to make saving versions for old versions that can't, but I don't know how to decompile/deobsficate anything. Thanks.

**Comment by nasko222** [comment]
4. You will not modify any of the source code of The Minecraft Recreation Project without the author's permission.

**Comment by nasko222** [comment]
PS: If you want to make saving mods, Make jarmods for MultiMC for the base versions of which remakes were built. Don't mod the remakes directly.

**Comment by davidlas** [comment]
On Fri, Sep 8, 2023 at 1:24 PM Nasko ***@***.***> wrote:

> PS: If you want to make saving mods, Make jarmods for MultiMC for the base
> versions. Don't mod the remakes
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nasko222/the-minecraft-recreation-project/discussions/157#discussioncomment-6950706>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBR3CU3OMIFABY5CPWTLILTXZNIDPANCNFSM6AAAAAA4QVVXRY>
> .
> You are receiving this because you authored the thread.Message ID:
> <nasko222/the-minecraft-recreation-project/repo-discussions/157/comments/6950706
> @github.com>
>
Sorry, I forgot about that. Would you happen to have the source code for
the non-alpha/beta versions? Or would I need to be a contributed? Also, if
you don’t mind me asking, I’d that in place mainly because of method (not
to sound to intrusive.) Thanks.

**Comment by nasko222** [comment]
Its fine, but sorry i dont keep the source code for the remakes. And I switched PCs 1 week ago

**Comment by davidlas** [comment]
Ok, so videos of with the versions is fine, tho?

**Comment by nasko222** [comment]
Yes

**Comment by davidlas** [comment]
And with acknowledgment, of course. I don’t understand the reasoning behind
it, but I cam respect that.

**Comment by nasko222** [comment]
The reason is people. I already have quite a lot of kids targeting the project.

**Comment by davidlas** [comment]
Targeting?

**Comment by nasko222** [comment]
Stealing or passing it off as real. I would like to have as minimal of these scenarios thats why the EULA exists, and if someone gets WAY too far I will contact my attorney

**Comment by nasko222** [comment]
Prime example: https://www.youtube.com/channel/UCGK6qMk0dmaJWMcNPMlxiSw/videos

**Comment by davidlas** [comment]
I see. Well, I will definitely make sure to credit you when I make my
polyworld. Not just cause the layer thing. If you don’t mind me asking one
last thing, do you know of a way to fix the terrain repopulation issue in
the (official) 0327? Like adding the chunk files as a schematic after
creating alpha chunks in the same spots. Thanks.

**Comment by nasko222** [comment]
I havent tinkered with mid-infdev, sorry I dont know. And thanks that youre gonna credit me. Enjoy your polyworld.

**Comment by davidlas** [comment]
Your’e welcome. I’ll start it in May for the 15th anniversary of the game,
but I need to finish downloading your remakes and fixing bugs like the
terrain bug before then.
